ext/web/lib.deno_web.d.ts

```diff
@@ -223,22 +231,38 @@ declare type EventListenerOrEventListenerObject =
   | EventListener
   | EventListenerObject;
 
-/** @category Events */
+/**  adds three more optional properties to those inherited from
```

Comment:
```suggestion
/**  adds three more optional properties to those inherited from
```
Missing start of sentence?